site/content/entry/yuzu-progress-report-sep-2020/index.md

```diff
@@ -0,0 +1,133 @@
+
+{{< imgs
+    "./menu.png| For now, just the menu and soundtrack sections! (Super Mario 3D All-Stars)"
+  >}}
+  
+## Vulkan changes
+
+`VK_KHR_timeline_semaphore` is a relatively recent Vulkan extension only supported by *desktop or laptop* GPUs. In the past, resources were being assigned with `VkFence` and freed when arbitrarily asked, leading to a possible small pileup. With `VK_KHR_timeline_semaphore`, each resource is tied to a periodically checked "GPU tick," and you only need to verify on which tick you are at the moment to know if that resource is now free.
+
+{{< imgs
+    "./vk.png| Our implementation is not that complex. This is just an example provided by Khronos"
+  >}}
+
+[Adding support for timeline semaphores on yuzu](https://github.com/yuzu-emu/yuzu/pull/4674) helps simplify the code a lot, improves VRAM use a small amount, and paves the way for the finalization of the `Texture Cache rewrite`. Thank you, [Rodrigo](https://github.com/ReinUsesLisp), for its addition! The result should be transparent for the user with no perceptible change besides a bit of VRAM savings.
+
+## Bug fixes and improvements
+
+A missing feature in the input rewrite was the ability to merge two separate single left and right Joy-Cons into a single dual Joy-Con, a function `Super Mario Odyssey` needs. [Morph](https://github.com/Morph1984) [implemented `MergeSingleJoyasDualJoy`](https://github.com/yuzu-emu/yuzu/pull/4629), so this is no longer a problem for Mario. He can return to work once again for nothing. Thirty-nine years and no rewards for the best hero the Mushroom Kingdom ever saw.
+
+A small PSA for our Linux users, the [Qt required version has been raised to `5.12.8`](https://github.com/yuzu-emu/yuzu/pull/4638) now. Thanks [Morph](https://github.com/Morph1984) for the update! We need those new Plasma functions.
+
+Users have been reporting input latency, so [Tobi](https://github.com/FearlessTobi) decided to [test some changes](https://github.com/yuzu-emu/yuzu/pull/4643) and see if the situation improves. To further mitigate this issue if you experience it, be sure to enable `Radeon Anti-Lag` or `Nvidia Ultra-Low Latency` in your respective driver control panel. Nothing makes you ragequit more than losing in Smash due to lag.
+
+Now [Gamecube controllers will be autoconfigured!](https://github.com/yuzu-emu/yuzu/pull/4618) Thanks to work done by [german77](https://github.com/german77), now the Gamecube Adapter controllers will show up as devices in the input window, and their buttons will be mapped automatically. Laziness is the best.
+
+`Clubhouse Games: 51 Worldwide Classics` now manages to boot and, with a save, load the board games. [Morph](https://github.com/Morph1984) managed this by [implemention the `Get/SetNpadHandheldActivationMode` function](https://github.com/yuzu-emu/yuzu/pull/4683). The game will render almost everything in black, for now, so don't consider it playable yet.
```

Comment:
Change `The game will render almost everything in black, for now, so don’t consider it playable yet.` to `However, the game will still render almost everything in black, so don't consider it playable yet.`

Reply:
Also change `[Morph](https://github.com/Morph1984) managed this by [implemention the `Get/SetNpadHandheldActivationMode` function]` to `[Morph](https://github.com/Morph1984) managed this by [implementing the `Get/SetNpadHandheldActivationMode` functions]